aspnetcore/fundamentals/openapi/samples/9.x/BuildTime/TestScripts/addProps.ps1

```diff
@@ -0,0 +1,17 @@
+# Adds OpenApiDocumentsDirectory to MyOpenApiTest.csproj
+# builds
+$env:ProgramName = "MyOpenApiTest"
+$env:Project = "$($env:ProgramName).csproj"
+$xml = [xml](Get-Content $env:Project)
+
+$newPropertyGroup = $xml.CreateElement("PropertyGroup")
+$newElement = $xml.CreateElement("OpenApiDocumentsDirectory")
+$newElement.InnerText = "./"
```

Comment:
@captainsafia run `/TestScripts/create.cmd` then run this script. I get the error `error : Missing required option '--project'.`